machine.go

```diff
@@ -204,6 +204,37 @@ func (cfg *Config) Validate() error {
 	return nil
 }
 
+func (cfg *Config) LoadSnapshotValidate() error {
```

Comment:
There seems to be slight inconsistencies with if the validation is done as apart of the handler of as a method of `Config`.

At the very least I think the method name should be verb first. e.g. `ValidateLoadSnapshot()`

Reply:
I see what you're saying, but I think this would still be the right place to validate these options. The other handlers are in their specified file because the only part of the config it validates is its own field. Validate and LoadSnapshotValidate both require multiple areas of the config to validate. So, I think this would be the right place to put it, and as no other function calls it, I don't really see an issue keeping it as is.